vyxal/transpile.py

```diff
@@ -130,7 +130,9 @@ def transpile_structure(struct: structure.Structure, indent: int) -> str:
         return res
     if isinstance(struct, structure.ForLoop):
         # TODO (user/ysthakur) make it work with multiple variables
-        var = struct.names[0] if struct.names else f"LOOP{secrets.token_hex(16)}"
+        var = (
+            struct.names[0] if struct.names else f"LOOP{secrets.token_hex(16)}"
```

Comment:
I'm sure some of us must have slightly different linting settings, because this kind of meaningless reformatting seems to happen a lot. Not a big deal, but discuss in chat?

Reply:
sure